docs/source/build_from_source.rst

```diff
@@ -27,6 +27,7 @@ depends through
     pip install -U nose
```

Comment:
pytest replaced nose. you can remove nose (it is deprecated/not compatible with 3.7+)